keyboard.vim

```diff
@@ -119,7 +119,7 @@ let g:leader_key_map.s = {
       \ 'name': '+search',
       \ 'g': ['Grepper',            'Find in directory (quickfix)'],
       \ 'f': [':Clap grep ',        'Find in directory (live)'],
-      \ 't': [':Clap tags',         'Find tags'],
+      \ 't': [':Clap tags coc',         'Find tags'],
```

Comment:
I'll fix this post-merge unless you beat me to it, but you forgot to re-align the dictionary (formatting).